### ISSUE TITLE
Make custom emojis visible to Voice Over

### ISSUE THREAD

**Issue by StephenRadachy** [metabolist/metatext#264]
**Describe the bug**
Currently when Voice Over reads a text selection with custom emojis, it will ignore them.

**To Reproduce**
1. Navigate to a post where a username has custom emoji in it using Voice Over
2. Notice that the custom emoji is not spoken

**Expected behavior**
Custom emojis should be announced

**Screenshots**
<img src="https://user-images.githubusercontent.com/5440238/204179719-24df7f58-75e7-4482-8468-b115824e045e.jpeg" width="300px" />

**(please complete the following information):**
 - Device: iPhone 14 Pro Max
 - OS: iOS 16.0.2
 - App Version: 1.7.0

**Comment by alexbbrown** [comment]
how about when custom emojis are used in posts-what do you get then?

**Comment by StephenRadachy** [comment]
> how about when custom emojis are used in posts-what do you get then?

Tested that out - same thing. They get ignored

**Comment by alexbbrown** [comment]
What about normal emojis?

**Comment by StephenRadachy** [comment]
Normal emojis work just fine
